index.js

```diff
@@ -6,6 +6,39 @@ import * as DID from '@ipld/dag-ucan/did'
 import { CarWriter } from '@ipld/car'
 import { getClient, checkPathsExist, filesize, readProof, filesFromPaths, uploadListResponseToString } from './lib.js'
 
+export async function accessClaim () {
+  const client = await getClient()
+  const delegations = await client.capability.access.claim()
+  for (const delegation of delegations) {
+    console.log('claimed delegation', JSON.stringify(delegation, null, 4))
+  }
+}
+
+/**
+ * @param {string} email
+ */
+export async function authorize (email) {
+  const client = await getClient()
+  /** @type {import('ora').Ora|undefined} */
+  let spinner
+  setTimeout(() => {
+    spinner = ora(`🔗 please click the link we sent to ${email} to authorize this device`).start()
+  }, 1000)
+  try {
+    await client.capability.access.authorize(email)
+  } catch (err) {
+    if (spinner) spinner.stop()
+    if (err.message.startsWith('Device already authorized')) {
```

Comment:
I'm not sure what produces this specific error, I don't think backend does. Do we need to error here ? If we do can we at least instruct user what can they do ? If they can just carry on, maybe we should not error and just say "agent" is authorized ?

Reply:
oh good question, will investigate, this might be a relic from copy-pasting `registerSpace`...

Reply:
deleted this